.doc_gen/metadata/api-gateway_metadata.yaml

```diff
@@ -189,21 +178,29 @@ api-gateway_GetRestApis:
                 - apigateway.rust.get_rest_apis
   services:
     api-gateway: {GetRestApis}
-api-gateway_DeleteRestApi:
-  title: Delete an &ABP; REST API using an &AWS; SDK
-  title_abbrev: Delete a REST API
-  synopsis: delete an &ABP; REST API.
+api-gateway_ListBasePathmappings:
+  title: List the base path mapping for a custom domain name in &ABP;
+  title_abbrev: List base path mapping
+  synopsis: list an &ABP; base path mapping.
   category:
   languages:
-    Java:
+    PHP:
       versions:
-        - sdk_version: 2
-          github: javav2/example_code/apigateway
+        - sdk_version: 3
+          github: php/example_code/apigateway
           sdkguide:
           excerpts:
             - description:
               snippet_tags:
-                - apigateway.java2.delete_api.main
+                - apigateway.php.list_base_path_mapping.complete
+  services:
+    api-gateway: {ListBasePathmappings}
```

Comment:
ListBasePathmappings -> ListBasePathMappings
(must match Smithy API casing for API ref to work)

Reply:
Fixed